src/config.erl

```diff
@@ -403,8 +403,9 @@ parse_ini_file(IniFile) ->
             ";" ++ _Comment ->
                 {AccSectionName, AccValues};
             Line2 ->
-                case re:split(Line2, "\s?=\s?", [{return, list}]) of
```

Comment:
would `"\s?(?<!\\\\)=\s?"` work here? (that is, split the string only on `=` when not immediately preceded by `\`.